Windows installer does not remove previous install

I tried to install OpenRefine 3.9-beta1 on a Windows machine which had a previous installation of OpenRefine, and it added the files from 3.9-beta1 in the directory where OpenRefine 3.8 was already installed, mixing up the files together.
This leads to a startup failure because conflicting versions of the same jars are present.

Removing OpenRefine (by deleting the folder in C:\Program Files) and reinstalling it solves the problem.

Perhaps our InnoSetup configuration could be adjusted for the installer to detect and clean up the previous install?

### To Reproduce
Steps to reproduce the behavior:
1. First, install OpenRefine 3.7.9 on Windows (with the installer)
2. Then, install OpenRefine 3.9-beta1 (also with the installer)

### Current Results

OpenRefine fails to start, giving the following logs:
```
HTTP ERROR 500 Butterfly Error Butterfly incurred in the following errors while initializing: java.lang.Exception: Failed to initialize module: core [edu.mit.simile.butterfly.ButterflyModuleImpl] at edu.mit.simile.butterfly.Butterfly.initializeModule(Butterfly.java:465) at edu.mit.simile.butterfly.Butterfly.configure(Butterfly.java:438) at edu.mit.simile.butterfly.Butterfly.init(Butterfly.java:295) at org.eclipse.jetty.servlet.ServletHolder$Wrapper.init(ServletHolder.java:1305) at org.eclipse.jetty.servlet.ServletHolder.initServlet(ServletHolder.java:633) at org.eclipse.jetty.servlet.ServletHolder.getServlet(ServletHolder.java:486) at org.eclipse.jetty.servlet.ServletHolder.prepare(ServletHolder.java:731) at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:524) at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:131) at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:578) at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122) at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:223) at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1570) at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:131) at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122) at org.eclipse.jetty.server.handler.gzip.GzipHandler.handle(GzipHandler.java:822) at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122) at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:223) at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1384) at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:176) at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:484) at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1543) at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:174) at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1306) at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:129) at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122) at com.google.refine.ValidateHostHandler.handle(ValidateHostHandler.java:93) at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122) at org.eclipse.jetty.server.Server.handle(Server.java:563) at org.eclipse.jetty.server.HttpChannel$RequestDispatchable.dispatch(HttpChannel.java:1598) at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:753) at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:501) at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:282) at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:314) at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:100) at org.eclipse.jetty.io.SelectableChannelEndPoint$1.run(SelectableChannelEndPoint.java:53) at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) at java.base/java.lang.Thread.run(Unknown Source) Caused by: org.mozilla.javascript.EvaluatorException: Java class "com.google.refine.grel.Parser" has no public instance field or method named "grelParser". (file:/C:/Program%20Files%20(x86)/OpenRefine/webapp/modules/core/MOD-INF/controller.js#342) at org.mozilla.javascript.DefaultErrorReporter.runtimeError(DefaultErrorReporter.java:79) at org.mozilla.javascript.Context.reportRuntimeError(Context.java:907) at org.mozilla.javascript.Context.reportRuntimeError(Context.java:963) at org.mozilla.javascript.Context.reportRuntimeErrorById(Context.java:914) at org.mozilla.javascript.JavaMembers.reportMemberNotFound(JavaMembers.java:849) at org.mozilla.javascript.NativeJavaClass.get(NativeJavaClass.java:102) at org.mozilla.javascript.ScriptableObject.getProperty(ScriptableObject.java:2023) at org.mozilla.javascript.ScriptRuntime.getObjectProp(ScriptRuntime.java:1672) at org.mozilla.javascript.ScriptRuntime.getObjectProp(ScriptRuntime.java:1667) at org.mozilla.javascript.gen.file__C__Program_20Files_20_x86__OpenRefine_webapp_modules_core_MOD_INF_controller_js_2._c_registerLanguages_5(file:/C:/Program%20Files%20(x86)/OpenRefine/webapp/modules/core/MOD-INF/controller.js:342) at org.mozilla.javascript.gen.file__C__Program_20Files_20_x86__OpenRefine_webapp_modules_core_MOD_INF_controller_js_2._c_init_8(file:/C:/Program%20Files%20(x86)/OpenRefine/webapp/modules/core/MOD-INF/controller.js:373) at org.mozilla.javascript.gen.file__C__Program_20Files_20_x86__OpenRefine_webapp_modules_core_MOD_INF_controller_js_2.call(file:/C:/Program%20Files%20(x86)/OpenRefine/webapp/modules/core/MOD-INF/controller.js) at org.mozilla.javascript.ContextFactory.doTopCall(ContextFactory.java:380) at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3868) at org.mozilla.javascript.gen.file__C__Program_20Files_20_x86__OpenRefine_webapp_modules_core_MOD_INF_controller_js_2.call(file:/C:/Program%20Files%20(x86)/OpenRefine/webapp/modules/core/MOD-INF/controller.js) at edu.mit.simile.butterfly.ButterflyModuleImpl.scriptInit(ButterflyModuleImpl.java:640) at edu.mit.simile.butterfly.ButterflyModuleImpl.init(ButterflyModuleImpl.java:94) at edu.mit.simile.butterfly.Butterfly.initializeModule(Butterfly.java:463) ... 38 more
```
<!-- What results occurred or were shown. -->

### Expected Behavior

Successful start

### Screenshots

See the mixes of library versions:
![Image](https://github.com/user-attachments/assets/77a4ae72-4e85-4ae1-a6c0-f55904fc24b0)

### Versions<!--  (please complete the following information)-->
 - Operating System: Windows 10
 - Browser Version: Edge
 - JRE or JDK Version: embedded

--- a/packaging/openrefine.iss
+++ b/packaging/openrefine.iss
@@ -66,6 +66,12 @@ Name: "ukrainian"; MessagesFile: "compiler:Languages\Ukrainian.isl"
 [Tasks]
 Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{cm:AdditionalIcons}"; Flags: unchecked
 
+; Make sure we delete any previous install before copying our files.
+; See https://github.com/OpenRefine/OpenRefine/issues/7122
+[InstallDelete]
+Type: filesandordirs; Name: "{app}\server"
+Type: filesandordirs; Name: "{app}\webapp"
+
 [Files]
 ; NOTE: Don't use "Flags: ignoreversion" on any shared system files
 Source: "{#MyProgramFiles}\*"; DestDir: "{app}"; Flags: ignoreversion recursesubdirs

